string id
string type
string time
string received_time
string received_cmd
r_pose_optional[] received_waypoints
string result
string[] errors
